pkg/common/telemetry/prometheus.go

```diff
@@ -49,6 +49,7 @@ func newPrometheusRunner(c *MetricsConfig) (sinkRunner, error) {
 	if runner.c.Host != "localhost" {
 		runner.log.Warnf("Agent is now configured to accept remote network connections for Prometheus stats collection. Please ensure access to this port is tightly controlled")
 	}
+	runner.log.Debug("Starting prometheus listener on %s:%d", runner.c.Host, runner.c.Port)
```

Comment:
At a minimum, this should use Debugf or the lines aren't going to be formatted properly. Ideally we use structured fields for the host and port:
```suggestion
	runner.log.WithFields(logrus.Fields{
		"host": runner.c.Host,
		"port": runner.c.Port,
	}).Debug("Starting prometheus listener")
```

Reply:
Info would make sense

Reply:
Also, I think the file needs to be fixed with gofmt/goimports.... there is some bad indentation on line 55 of prometheus.go